docs/root/configuration/http_filters/fault_filter.rst

```diff
@@ -89,5 +100,7 @@ owning HTTP connection manager.
 
   delays_injected, Counter, Total requests that were delayed
   aborts_injected, Counter, Total requests that were aborted
+  faults_overflow, Counter, Total number of faults that were not injected due to overflowing the :ref:`max_active_faults <envoy_api_field_config.filter.http.fault.v2.HTTPFault.max_active_faults>` setting
```

Comment:
nit: faults_omitted ? overflow and faults dont seem to click immediately

Reply:
overflow is how we talk about circuit breakers currently, so that is why I named it this. I don't feel strongly if you feel omitted is better. WDYT?